kolibri_explore_plugin/assets/src/modules/topicsRoot/handlers.js

```diff
@@ -217,6 +220,7 @@ export function showChannels(store) {
         });
     },
     error => {
+      ChannelResource.useContentCacheKey = true;
       store.dispatch('handleApiError', error);
       return error;
     }
```

Comment:
I think this could be clearer if we used `Promise.prototype.finally()`:

```js
    error => {
      store.dispatch('handleApiError', error);
      return error;
    }
  ).finally(
    () => ChannelResource.useContentCacheKey = true;
  );
```

Reply:
Ah absolutely @dylanmccall , this is better for what I meant, to reset the value anyways. Thanks, fixing and merging.

Reply:
Unfortunately the pre-commit hook changed the spacing of the whole function with the `finally`. Merging!